__mocks__/next-auth/nextAuthAPI.mock.ts

```diff
@@ -0,0 +1,139 @@
+import { ReactChildren } from 'react'
+import { SubmissionStatus } from '../../graphql'
+import dummySessionData from '../../__dummy__/sessionData'
+
+const mockSessionProvider = jest
+  .fn()
+  .mockImplementation(({ children }: { children: ReactChildren }) => children)
+const mockSignOut = jest.fn()
+const mockSignIn = jest.fn()
+const useSessionReturn = {
```

Comment:
When mocking data that is used accross components, please [object freeze](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze) it.